src/EluvioLive.js

```diff
@@ -353,6 +353,150 @@ class EluvioLive {
+    );
+
+    let errors = [];
+
+    let tenantAdminAddr = await this.client.CallContractMethod({
+      contractAddress: tenantAddr,
+      abi: JSON.parse(abi),
+      methodName: "groupsMapping",
+      methodArgs: ["tenant_admin", 0],
+      formatArguments: true,
+    });
+    tenantInfo["tenant_admin_address"] = tenantAdminAddr;
+    
+    //Content admins group might not exist for the tenant with this tenantId due to legacy reasons.
+    //Running ./elv-live tenant-fix to update this tenant.
+    let contentAdminAddr;
+    try {
+      contentAdminAddr = await this.client.CallContractMethod({
+        contractAddress: tenantAddr,
+        abi: JSON.parse(abi),
+        methodName: "groupsMapping",
+        methodArgs: ["content_admin", 0], 
+        formatArguments: true,
+      });
+    } catch (e) {
+      contentAdminAddr = null;
+      errors.push("missing content admins");
+    }
+    tenantInfo["content_admin_address"] = contentAdminAddr;
+
+    if (errors.length != 0) {
+      tenantInfo["errors"] = errors;
+    }
+
+    return tenantInfo;
+  }
+
+  /**
+   * Create a new content admin group corresponding to this tenant.
+   * @param {string} tenantId - The ID of the tenant (iten***)
+   * @returns {string} Content Admin Group's address
+   */
+  async TenantCreateContentAdmin({ tenantId }) {
+    const tenantOwner = await this.client.authClient.Owner({id: tenantId});
+    if (tenantOwner.toLowerCase() != this.client.signer.address.toLowerCase()) {
+      throw Error("Content Admin must be created by the owner of tenant " + tenantId);
+    }
+
+    let elvAccount = new ElvAccount({configUrl:this.configUrl, debugLogging: this.debug});
+    elvAccount.InitWithClient({elvClient:this.client});
+
+    let accountName = await this.client.userProfileClient.UserMetadata({
+      metadataSubtree: "public/name"
+    });
+  
+    let contentAdminGroup = await elvAccount.CreateAccessGroup({
+      name: `${accountName} Content Admins`,
+    });
+
+    await elvAccount.AddToAccessGroup({
+      groupAddress: contentAdminGroup.address,
+      accountAddress: this.client.signer.address.toLowerCase(),
+      isManager: true,
+    });
+
+    return contentAdminGroup.address;
+  }
+
+  /**
+   * Set a new content admin corresponding to this tenant.
+   * @param {string} tenantId - The ID of the tenant (iten***)
+   * @param {string} contentAdminsAddress - Address of content admin we want to add.
+   */
+  async TenantAddContentAdmin({ tenantId, contentAdminsAddress }) {
+    const abi = fs.readFileSync(
+      path.resolve(__dirname, "../contracts/v3/BaseTenantSpace.abi")
+    );
+
+    const tenantAddr = Utils.HashToAddress(tenantId);
+
+    let contentAdmin;
+    try {
+      contentAdmin = await this.client.CallContractMethod({
+        contractAddress: tenantAddr,
+        abi: JSON.parse(abi),
+        methodName: "groupsMapping",
+        methodArgs: ["content_admin", 0],
+        formatArguments: true,
+      });
+    } catch (e) {
```

Comment:
Call cannot override gasLimit error is thrown when the content admin group doesn't exist. I am wondering if there is better way to check this.